back/engines/free/user_confirmation/config/initializers/constants.rb

```diff
@@ -0,0 +1 @@
+USER_CONFIRMATION_CODE_PATTERN = /\A[0-9]{4}\z/i.freeze
```

Comment:
Could perhaps be a good topic for the next code discussion meeting? If we stick with this, I guess we might want to consider moving in other constants here.

Reply:
This isn't actually a bid deal it could be stored in either the user patch or here, for this case it could make sense to have it there, but other than that I think its useful to have them all in once place so we can configure/change to env variables if we feel its useful